docker/site.conf

```diff
@@ -0,0 +1,4 @@
+name: site
```

Comment:
Maybe rename this file to `example-site.conf`?

Reply:
Well, eventually I hope this grows into the real thing. Yeah, it's not very usable at the moment, but it's not intended to be an example.